cmd/update_test.go

```diff
@@ -141,4 +141,46 @@ var _ = Describe("Update", func() {
 			Eventually(session).Should(gexec.Exit(0))
 		})
 	})
+
+	Describe("When Github returns a 403 error", func() {
+		BeforeEach(func() {
+			command = exec.Command(pathCLI,
+				"update", "check",
+				"--github-api", tempSettings.TestServer.URL(),
+			)
+
+			tempSettings.TestServer.Reset()
+			tempSettings.TestServer.AppendHandlers(
+				ghttp.CombineHandlers(
+					ghttp.VerifyRequest("GET", "/repos/CircleCI-Public/circleci-cli/releases"),
+					ghttp.RespondWith(http.StatusForbidden, []byte("Forbidden")),
+				),
+			)
+		})
+
+		It("should print a helpful error message & exit 255", func() {
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ShouldNot(HaveOccurred())
+			Eventually(session).Should(gexec.Exit(255))
+
+			// TODO: This should exit with error status 1, since 255 is a
+			// special error status for: "exit status outside of range".
```

Comment:
What is stopping us from following through with this `TODO`? If there's a good reason to postpone it, it would be useful to put an explanation in the comment here as well.

Reply:
I extended the comment to add some more context, but the problem is essentially all commands that return an error get the same status code: https://github.com/CircleCI-Public/circleci-cli/blob/5896baa95dad1b66f9c4a5b0a14571717c92aa55/cmd/root.go#L38

Reply:
I'm just guessing here, but is `-1` syntactic sugar for 255? If so, I wonder if there's any time we actually want to return 255 instead of 1.